### ISSUE TITLE
Nitrogen Deposition file for SSP534OS

### ISSUE THREAD

**Issue by olyson** [ESCOMP/CTSM#1131]
Per Mike Mills, the NorESM group requested that we generate forcing files from WACCM output for SSP5-3.4OS (b.e21.BWSSP534oscmip6.f09_g17.CMIP6-SSP5-3.4OS-WACCM.001).  Thus, Mike requested that I generate a nitrogen deposition file from this simulation.
It is here:
/glade/p/cgd/tss/people/oleson/CLM5_ndep/fndep_clm_SSP534OS_b.e21.BWSSP534oscmip6.f09_g17.CMIP6-SSP5-3.4OS-WACCM.001_2039-2101_monthly_0.9x1.25_c200831

The WACCM simulation was initialized at year 2040 and ran through year 2100, so I've created a file running from 2039-2101.
I'm not sure if this is needed/required by any other group (including CESM), but perhaps this should be placed in inputdata with the other ndep files.

I wasn't able to create a corresponding aerosol deposition file for land-only simulations because coupler history output wasn't saved in the WACCM simulation.  Alternatively, the file could be generated from a CAM simulation, but that simulation doesn't exist at present.

**Comment by ekluzek** [comment]
@Katetc here's info. on the SSP5-3.4 OS ndep file.

**Comment by ekluzek** [comment]
OK, we copied the file from @olyson directory to main inpudata:

lnd/clm2/ndepdata/fndep_clm_SSP534osext_b.e21.BWSSP534osextcmip6.f09_g17.CMIP6-SSP5-3.4OSext-WACCM.001_2100-2300_monthly_0.9x1.25_c200624.nc

This was from:

/glade/p/cgd/tss/people/oleson/CLM5_ndep/fndep_clm_SSP534osext_b.e21.BWSSP534osextcmip6.f09_g17.CMIP6-SSP5-3.4OSext-WACCM.001_2100-2300_monthly_0.9x1.25_c200624.nc

**Comment by ekluzek** [comment]
OK, we need to stitch the historical period and 2015-2100 period onto the same file, so a continuous simulation can be done rather than having to do this in segments. Based on #1213 it should switch at 2015 as well as 2100.

**Comment by Katetc** [comment]
Hi Eric! Just a note that the SSP5-34os scenario is a little different. It actually branches from the SSP5-85 scenario at year 2040. Our land ice simulation will go from year 2040 to 2300. So, we don't need data before year 2040.
Thanks!

**Comment by ekluzek** [comment]
If we include the historical period the segments needed would be this sequence of four files: From @Katetc, it sounds like all that's required here is just the last two.

The historical segment is: lnd/clm2/ndepdata/fndep_clm_hist_b.e21.BWHIST.f09_g17.CMIP6-historical-WACCM.ensmean_1849-2015_monthly_0.9x1.25_c180926.nc
A future segment that's needed is from SSP5-8.5: lnd/clm2/ndepdata/fndep_clm_SSP585_b.e21.BWSSP585cmip6.f09_g17.CMIP6-SSP5-8.5-WACCM.001_2014-2101_monthly_0.9x1.25_c190211.nc
The SSP5-3.4 future segment is then:
lnd/clm2/ndepdata/fndep_clm_SSP534OS_b.e21.BWSSP534oscmip6.f09_g17.CMIP6-SSP5-3.4OS-WACCM.001_2039-2101_monthly_0.9x1.25_c200831.nc
The final extension segment is then:
lnd/clm2/ndepdata/fndep_clm_SSP585ext_b.e21.BWSSP585extcmip6.f09_g17.CMIP6-SSP5-8.5ext-WACCM.001_2100-2300_monthly_0.9x1.25_c200623.nc

**Comment by Katetc** [comment]
Yes, we only need the last two files, because the first two sections are already in the SSP5-85 simulation (which is done). For the last file, we want the one with the BWSSP534osextcmip6 compset. I think it is this one: lnd/clm2/ndepdata/fndep_clm_SSP534osext_b.e21.BWSSP534osextcmip6.f09_g17.CMIP6-SSP5-3.4OSext-WACCM.001_2100-2300_monthly_0.9x1.25_c200624.nc

**Comment by ekluzek** [comment]
Yes, sorry you are right, I corrected it above. I compared the 2101 segment on the two files and they are different, while 2100 is identical. So I'll use the first file up to 2099, and the full last file. That way 2100 is still identical, but we use the new file for 2101.

Here's the new resulting file, which I'm archiving into svn inpudata:

lnd/clm2/ndepdata/fndep_clm_SSP534osext_b.e21.BWSSP534osextcmip6.f09_g17.CMIP6-SSP5-3.4OSext-WACCM.001_2100-2300_monthly_0.9x1.25_c200624.nc
